x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/index.tsx

```diff
@@ -135,7 +135,7 @@ const StepRuleActionsComponent: FC<StepRuleActionsProps> = ({
                 }}
               />
               <UseField
-                path="kibanaSiemAppUrl"
+                path="kibana_siem_app_url"
```

Comment:
Should this have been changed to snake case? The rest of the form fields use camelcase, and are converted to snake case for HTTP in the `format*` functions in rules/create/helpers.

Reply:
No it should not have changed. I did a sloppy job of replacement and then test without reading my own code.

I won't do such a bad job again. Thank you for taking the time to catch this.